style.css

```diff
@@ -116,3 +116,348 @@
   gap: 19px;
   align-items: baseline;
 }
+
+#work-container {
+  display: grid;
+  flex-direction: column;
+  gap: 20px;
+  padding-top: 12px;
+  background-color: #fff;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: auto;
+  gap: 10px;
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  margin: 50px auto;
+  padding: 16px 0;
+  position: static;
+}
```

Comment:
- [ ] As the previous reviewer said that you should make it display: flex. Please make the styling according to the image below :+1:
  
![inner-container-flex](https://user-images.githubusercontent.com/93039359/176935235-94f0d790-402c-4e82-b6f6-007a1d181613.png)

Reply:
@HAMMAS-SALEEM Thank you very much for this.